examples/with-temporal/README.md

```diff
@@ -0,0 +1,93 @@
+> —[Guillermo Rauch](https://twitter.com/rauchg/status/1316808665370820609)
+
+**Table of Contents**
+
+- [Starter project](#starter-project)
+- [Deploy your own](#deploy-your-own)
+  - [Web server](#web-server)
+  - [Worker](#worker)
+  - [Temporal Server](#temporal-server)
+- [How to use](#how-to-use)
+
+## Starter project
+
+This is a starter project for creating resilient Next.js applications with [Temporal](https://temporal.io/). Whenever our [API routes](https://nextjs.org/docs/api-routes/introduction) need to do any of the below, we can greatly increase our code's fault tolerance by using Temporal:
+
+- Perform a series of network requests (to a database, another function, an internal service, or an external API), any of which may fail (Temporal will automatically set timeouts and retry, as well as remember the state of execution in the event of power loss)
+- Do something that takes longer than 5 or 30 seconds (Vercel's serverless function execution [time limit](https://vercel.com/docs/platform/limits) for Hobby or Enterprise accounts, respectively)
+- Do something after a certain amount of time, like an hour or a month (or periodically—once an hour, or once a month)
+
+The starter project has this logic flow:
+
+- Load [`localhost:3000`](http://localhost:3000)
+- Click the “Create order” button
+- The click handler POSTs a new order object to `/api/orders`
+- The serverless function at `pages/api/orders/index.ts`:
+  - Parses the order object
+  - Tells Temporal Server to start a new Order Workflow, and passes the user ID and order info
+  - Waits for the Workflow result and returns it to the client
+- Temporal Server puts Workflow tasks on the `orders` task queue
+- The Worker executes chunks of Workflow and Activity code. The Activity code logs:
+
+```
+Reserving 2 of item B102
+Charging user 123 for 2 of item B102
+```
+
+- The Workflow completes and Temporal Server sends the result back to the serverless function, which returns it to the client, which alerts the result.
+
+Here is the Temporal code:
+
+- The Workflow: `temporal/src/workflows/order.ts`
+- The Activites: `temporal/src/activities/{payment|inventory}.ts`
+
+## Deploy your own
+
+### Web server
+
+The Next.js server can be deployed using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-temporal&project-name=with-temporal&repository-name=with-temporal)
```

Comment:
Could this section follow the same format as the other example READMEs?

Reply:
Do you mean just changing the first line language to "Deploy the example using Vercel" to match https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples? I removed the StackBlitz `## Preview` section because it wouldn't work without Temporal Server & Worker.